tests/testthat/test-prophet.R

```diff
@@ -0,0 +1,663 @@
+     options$dependent <- "contNormal"
+     options$time <- "dateDay"
+     options$capacity <- "contGamma"
+     options$minimum <- "contNarrow"
+     options$historyIndicator <- "histIdx"
+     options$mcmcSamples <- 10
+     options$growth <- "logistic"
+     options$predictionSavePath <- ""
+     set.seed(1)
+     results <- runAnalysis("Prophet", "prophetTest.csv", options)
+     table <- results[["results"]][["prophetMainContainer"]][["collection"]][["prophetMainContainer_prophetTable"]][["data"]]
+     jaspTools::expect_equal_tables(table,
+          list(20, -7.29654434885841, -7.1818644571539, "Growth rate (k)", 1,
+                0.0843752327706313, 20, -7.06528194891996, 20, -0.148227664168065,
+                -0.00937252121369772, "Offset (m)", 1, 0.0963163677055294, 20,
+                0.1112940095092, 20, 0.397770975592959, 0.462882463647365, "Residual variance (sigma)",
+                1, 0.0830358930795562, 20, 0.598905146042098))
+})
+
+test_that("Posterior Summary Table results match (constant-logistic)", {
+  options <- analysisOptions("Prophet")
+  options$dependent <- "contNormal"
+  options$time <- "dateDay"
+  options$constantCapacity <- 10
+  options$constantMinimum <- 0.1
+  options$mcmcSamples <- 10
+  options$growth <- "logistic"
+  options$predictionSavePath <- ""
+  set.seed(1)
+  results <- runAnalysis("Prophet", "prophetTest.csv", options)
+  table <- results[["results"]][["prophetMainContainer"]][["collection"]][["prophetMainContainer_prophetTable"]][["data"]]
+  jaspTools::expect_equal_tables(table,
+    list(20, -3.59423543336109, -1.60259694991599, "Growth rate (k)", 1,
+       1.18253175777653, 20, -0.810313940048772, 20, -3.35660841596216,
+       -3.11650941860635, "Offset (m)", 1, 0.157198845587502, 20, -2.96206374129486,
+       20, 0.354910105713339, 0.480225356561229, "Residual variance (sigma)",
+       1, 0.0771605826106158, 20, 0.555518365514962))
+})
+
+test_that("Parameter Estimates Table results match (MAP)", {
+  options <- analysisOptions("Prophet")
+  options$dependent <- "contNormal"
+  options$time <- "dateDay"
+  options$predictionIntervalSamples <- 10
+  options$estimation <- "map"
+  options$predictionSavePath <- ""
+  set.seed(1)
+  results <- runAnalysis("Prophet", "prophetTest.csv", options)
+  table <- results[["results"]][["prophetMainContainer"]][["collection"]][["prophetMainContainer_prophetTable"]][["data"]]
+  jaspTools::expect_equal_tables(table,
+    list(-0.164828536217365, 0.0290249527327304, 0.315932056357582))
+
+  options$capacity <- "contGamma"
+  options$minimum <- "contNarrow"
+  options$historyIndicator <- "histIdx"
+  options$growth <- "logistic"
+  set.seed(1)
+  results <- runAnalysis("Prophet", "prophetTest.csv", options)
+  table <- results[["results"]][["prophetMainContainer"]][["collection"]][["prophetMainContainer_prophetTable"]][["data"]]
+  jaspTools::expect_equal_tables(table,
+    list(-2.50859195109382, -2.31656506358638, 0.32805219661169))
```

Comment:
These test values fail on my computer, are you sure they are correct?

Reply:
Hmm, I checked them again and they seem to be correct! The tests succeed on my computer... could it be a Windows/Mac thing again?